src/main/scala/io/keen/client/scala/Client.scala

```diff
@@ -361,6 +366,23 @@ trait Writer extends AccessLevel {
    */
   val writeKey: String = settings.writeKey.getOrElse(throw MissingCredential("Write key required for Writer"))
 
+  // constants
+  val MinSendInterval: Integer = 60
+  val MaxSendInterval: Integer = 3600
+
+  // initialize and configure our local event store queue
+  val batchSize: Integer = settings.batchSize.getOrElse(500)
+  val batchTimeout: Integer = settings.batchTimeout.getOrElse(5)
```

Comment:
Just my opinionated approach here, but I like all defaults to be in the `reference.conf`, not hardcoded. This way `reference.conf` is the authoritative documentation. They can have defaults with environment variable overrides allowed with the `${?VAR_NAME}` syntax.